app/components/primer/beta/flash.rb

```diff
@@ -8,21 +8,15 @@ class Flash < Primer::Component
 
       # Optional action content showed on the right side of the component.
       #
-      # @param system_arguments [Hash] <%= link_to_system_arguments_docs %>
-      renders_one :action, lambda { |**system_arguments|
-        deny_tag_argument(**system_arguments)
-        system_arguments[:tag] = :div
-        system_arguments[:classes] = class_names(system_arguments[:classes], "flash-action")
-
-        Primer::BaseComponent.new(**system_arguments)
-      }
+      # @param system_arguments [Hash] The same arguments as <%= link_to_component(Primer::Beta::Button) %>.
+      renders_one :action, Primer::Beta::Button
```

Comment:
I saw a few cases using things other than buttons in the `with_action` https://github.com/search?q=repo%3Agithub%2Fgithub+.with_action&type=code

Reply:
What if we rename it to `Banner` 😛